website/markdown/docs/building-at-scale/caching.mdx

```diff
@@ -27,6 +27,12 @@ tuist cache warm
 
 <ArgumentsTable
   args={[
+    {
+      long: '`--profile`',
+      short: '`-P`',
+      description: 'The name of the profile to be used when warming up the cache.',
```

Comment:
I think the profile deserves its own section on this page. If I just read the website's documentation, I have no idea what a profile actually is

Reply:
I added the section 👍